.goreleaser.yml

```diff
@@ -168,3 +178,54 @@ brews:
         <string>#{var}/log/swarm-bee/bee.log</string>
       </dict>
       </plist>
+
+dockers:
+- image_templates:
+  - "ethersphere/bee:{{ .Version }}-amd64"
+  use_buildx: true
+  ids:
+    - linux
+  goarch: amd64
+  dockerfile: Dockerfile.goreleaser
+  build_flag_templates:
+  - "--platform=linux/amd64"
+  - "--pull"
+  - "--label=org.opencontainers.image.created={{.Date}}"
+  - "--label=org.opencontainers.image.title={{.ProjectName}}"
+  - "--label=org.opencontainers.image.revision={{.FullCommit}}"
+  - "--label=org.opencontainers.image.version={{.Version}}"
```

Comment:
Linux amd64 is also in build without docker. This looks to me like a redundant part.

Reply:
All binaries are built outside the docker, we are using `Dockerfile.goreleaser` that just copies already built binaries. This is done by [official docs](https://goreleaser.com/customization/docker/) first `Warning` section